app/controllers/courses_controller.rb

```diff
@@ -15,10 +14,9 @@ def create
       redirect_to course_show_path(@course.id)
       return
     else
-      render 'new'
-      return
+      render 'new' and return
```

Comment:
Use `&&` instead of `and`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.